apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1309,6 +1323,34 @@ block_txs_list_by_hash_invalid_range(_Config) ->
         ]),
     ok.
 
+%% ============================================================
+%% Websocket tests 
+%% ============================================================
+
+ws_get_genesis(_Config) ->
+    {Host, Port} = ws_host_and_port(),
+    {ok, ConnPid} = ?WS:start(Host, Port),
+    ?WS:send(ConnPid, chain, get, #{height => 0}),
```

Comment:
Could you please ensure this is [documented](https://github.com/aeternity/protocol/blob/68b27350f157e05c6930d037a626695e6a4fad3b/epoch/api/README.md#websocket-api-definition) ?
(It may require moving out of the oracle page some more generic portions.)